File: src/sdk-reference/js/6/collection/get-mapping/snippets/get-mapping.js
try {
  const mapping = await kuzzle.collection.getMapping('nyc-open-data', 'yellow-taxi');
  console.log(mapping);
  /*
    {
      dynamic: 'false',
      _meta: {
        area: 'Panipokhari
      },
      properties: {
        license: { type: 'keyword' },
        driver: {
          properties: {
            name: { type: 'keyword' },
            curriculum: { type: 'text' }
          }
        }
      }
    }
  */

  console.log('Success');
} catch (error) {
  console.error(error.message);
}

File: src/sdk/cpp/1/controllers/collection/get-mapping/snippets/get-mapping.cpp
try {
  std::string mapping = kuzzle->collection->getMapping("nyc-open-data", "yellow-taxi");

  std::cout << mapping << std::endl;
  // {"properties":{"license":{"type":"keyword"},"driver":{"properties":{"name":{"type":"keyword"},"curriculum":{"type":"text"}}}}}
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
